__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const app = require('../app.js');
+const request = require('supertest');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/index');
+const db = require('../db/connection');
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe('/api/categories', () => {
+  it('200: GET responds with an array of all cateory objects with the properties slug and description', () => {
+    return request(app)
+      .get('/api/categories')
+      .expect(200)
+      .then(({ body }) => {
+        const { categories } = body;
+        expect(categories).toBeInstanceOf(Array);
+        expect(categories).toHaveLength(4);
+        categories.forEach((category) => {
+          expect(category).toMatchObject({
```

Comment:
I like this test assertion, using a matcher like toMatchObject keeps it flexible so if more properties were added in the future, you'd be safe from this test failing 😄